### PR TITLE
Blockbase: initial dotorg upload version

### PR DESCRIPTION
**Pull request by jffng**
This is a version of Blockbase that adds the required function calls and files to get it to pass the automated theme check.

The requirements to get Mayland-blocks and Seedlet-blocks have also been added to this branch.

**To test**

- Check out this PR
- From the root themes folder, run `./package-dotorg.sh blockbase`
- Locate the generated zip within the blockbase folder. Upload and activate it
- Install and activate the automated [theme check plugin](https://wordpress.org/plugins/theme-check/)
- From wp-admin, visit Appearance > Theme Check and run the theme check
- Verify no Required errors are logged: 

Screen Shot 2021-05-25 at 1 58 21 PM

### CODE REVIEW

File: blockbase/footer.php [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?php
+/**
+ * Empty comments file to pass theme check
```
Comment (MaggieCabrera) [inline blockbase/footer.php]: I think this text doesn't go here :D

**Comment by MaggieCabrera** [comment]
Isn't Blockbase essentially a block theme (even though its children can be universal, Blockbase is only using block templates)? Are things like header.php needed for block themes too? TT1 blocks doesn't have it.

**Comment by MaggieCabrera** [comment]
Carolina's post about submitting block themes to dotorg: p29gdg-5ib-p2

File: blockbase/footer.php (hunk 1 above)
Comment (jffng) [inline blockbase/footer.php]:
```suggestion
 * Empty footer file to pass theme check
```

**Comment by scruffian** [comment]
Should this be merged?

**Comment by pbking** [comment]
This should not be merged.

File: mayland-blocks/functions.php [hunk 2]
```diff
@@ -21,3 +26,13 @@ function mayland_blocks_scripts() {
 	wp_enqueue_style( 'mayland-blocks-styles', get_stylesheet_directory_uri() . '/assets/theme.css', array( 'blockbase-ponyfill' ), wp_get_theme()->get( 'Version' ) );
 }
 add_action( 'wp_enqueue_scripts', 'mayland_blocks_scripts' );
+
+/**
+ * This function only exists to pass the theme-check.
+ * It does not do anything.
+ * Borrowed from aristath/q: https://themes.trac.wordpress.org/browser/q/0.1/functions.php?rev=137308
+ */
+function mayland_blocks_theme_check_dummy_calls() {
```
Comment (scruffian) [inline mayland-blocks/functions.php]:
```suggestion
function seedlet_blocks_theme_check_dummy_calls() {
```

**Review by scruffian — APPROVED** [review]
LGTM

**Comment by scruffian** [comment]
Sorry I just realised this shouldn't be merged.

**Comment by scruffian** [comment]
Reverted in #4038

**Pull request merged**